test/extended/core.sh

```diff
@@ -49,8 +49,8 @@ SKIP_TESTS=(
   Kibana                  # Not installed
   DNS                     # Can't depend on kube-dns
   kube-ui                 # Not installed by default
-  #DaemonRestart           # Experimental mode not enabled yet
-  #"Daemon set"            # Experimental mode not enabled yet
+  DaemonRestart           # Experimental mode not enabled yet (comment when supported)
```

Comment:
Instead of saying "comment when supported", it's better just to remove these 2 lines from this list once the tests are passing.